.nais/naiserator-prod.yaml

```diff
@@ -67,6 +67,8 @@ spec:
         diskAutoresize: true
   envFrom:
     - secret: ispersonoppgave-serviceuser
+  kafka:
```

Comment:
Erstatter dette eksisterende Kafka-topics? Kan i få fall env-vars for onprem-Kafka nedenfor fjernes?

Reply:
Den erstatter ikke alle nei, vi har fremdeles en onprem kafka-topic for å lese LPS oppfølgingsplaner fra syfooppfolgingsplanservice. Men fant en annen feil i en test pga spørsmålet ditt, så det er rettet opp nå :)!